src/Kunstmaan/AdminBundle/EventListener/AdminLocaleListener.php

```diff
@@ -75,7 +75,7 @@ public function onKernelRequest(GetResponseEvent $event)
      *
      * @return bool
      */
-    private function isAdminToken(TokenInterface $token, $providerKey)
+    private function isAdminToken(TokenInterface $token = null, $providerKey)
```

Comment:
If you assign a default value, you should revert the order of the parameters as well ...

http://php.net/manual/en/functions.arguments.php#example-171